src/Action/Iterator.php

```diff
@@ -53,6 +53,47 @@ public function where($field, $value)
         return $this;
     }
 
+    /**
+     * Applies FilterIterator condition imitating the sql LIKE operator - $field LIKE %$value% | $value% | %$value
+     *
+     * @param string $field
+     * @param string $value
+     *
+     * @return $this
+     */
+    public function like($field, $value)
+    {
+        $this->generator = new \CallbackFilterIterator($this->generator, function ($row) use ($field, $value) {
+
+            // skip row. does not have field at all
+            if (!isset($row[$field])) {
+                return false;
+            }
+
+
+            $clean_value = str_replace('%', '', $value);
+            // the row field exists check the position of th "%"(s)
+            switch ($value) {
+                // case "%str%"
+                case substr_count($value,'%')==2:
```

Comment:
Better use `substr($value, -1, 1) == '%' && substr($value, 0, 1) == '%'` to support '%' inside $value